docs/man-openmpi/man3/MPI_T_event_callback_get_info.3.rst

```diff
@@ -0,0 +1,55 @@
+.. _mpi_t_event_callback_get_info:
+
+
+MPI_T_event_callback_get_info
+=============================
+
+.. include_body
+
+:ref:`MPI_T_event_callback_get_info` |mdash| Returns a new info object containing the hints of the callback function registered for the callback safety level specified by cb_safety of the event-registration handle associated with event_registration.
+
+SYNTAX
+------
+
+
+C Syntax
+^^^^^^^^
+
+.. code-block:: c
+
+   #include <mpi.h>
+
+   int MPI_T_event_callback_get_info(MPI_T_event_registration event_registration,
+                                     MPI_T_cb_safety cb_safety, MPI_Info *info_used)
+
+
+INPUT PARAMETERS
+----------------
+* ``event_registration``: Event registration
+* ``cb_safety``: maximum callback safety level
+
+OUTPUT PARAMETERS
+-----------------
+
+* ``info_used``: Info argument
+
+DESCRIPTION
+-----------
+
+:ref:`MPI_T_event_callback_get_info` returns a new info object containing the
+hints of the callback function registered for the callback safety level specified by `cb_safety` of
```

Comment:
Nit: I don't know if we care, because I took a quick look through some other man pages and I don't think we're always consistent -- but shouldn't this be 

```
``cb_safety``
```

Not

```
`cb_safety`
```

Same goes for all other references to parameters in the description / text of the man page.